srcpkgs/clamav/template

```diff
@@ -58,11 +59,16 @@ post_install() {
 	sed -i ${PKGDESTDIR}/etc/freshclam.conf \
 		-e "s;^Example$;# Example;" \
 		-e "s;#DatabaseDirectory.*;DatabaseDirectory /var/lib/_${pkgname};"
+
+	# Remove clamav-config from main package
+	rm ${PKGDESTDIR}/usr/bin/clamav-config
 }
+
 clamav-devel_package() {
 	depends="${sourcepkg}>=${version}_${revision}"
 	short_desc+=" - development files"
 	pkg_install() {
+		vbin clamav-config
```

Comment:
Use `vmove usr/bin/clamav-config` instead ;)

Reply:
Fixed on the most recent push.